filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -32,6 +32,16 @@ ebb0(v0: i64, v1: i64, v2: i64, v3: i64, v4: i64):
 }
 ; check: function %five_args(i64 [%rcx], i64 [%rdx], i64 [%r8], i64 [%r9], i64 [32], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
 
+; check that we preserve xmm6 and above if we're using them locally
+function %float_callee_saves(f64, f64, f64, f64) windows_fastcall {
+ebb0(v0: f64, v1: f64, v2: f64, v3: f64):
+[-, %xmm6]  v4 = fadd v0, v1      ; bin: 66 0f db f3
```

Comment:
The `bin` expression here is wrong, but I've not gotten sufficiently far in this test to need to fix that yet. An earlier (more cumbersome) approach to preserving SIMD registers confirmed that this does get far enough to try generating stores to preserve values though, so that's a start.

Reply:
It appears that `filetests` doesn't actually test `bin` expressions (anymore?). This continued expressing the wrong bytes even while the test passed.

I thought I saw an incorrect `bin` in a different filetest, but wrote it off at the time. I'll file a followup issue for this.